apps/random_pipeline/build_pipelines.sh

```diff
@@ -93,16 +96,19 @@ for ((p=$((FIRST+1));p<$((FIRST+PIPELINES+1));p++)); do
     benchmark_time=$($PIPELINE_DIR/bench --estimate_all --benchmarks=all)
     echo $benchmark_time | tee -a $LOGFILE
     t=$(echo $benchmark_time | head -1 | awk '{print $8}')
-    id=$(printf "%04d%02d%02d%02d%02d" $P $STAGES $s $HL_RANDOM_DROPOUT $HL_BEAM_SIZE)
-    echo Producing featurization for ID: $id | tee -a $LOGFILE
-    $ADAMS2019_DIR/featurization_to_sample $PIPELINE_DIR/random_pipeline.featurization $t onnx $id $PIPELINE_DIR/random_pipeline.sample
+    pid=$(printf "%04d%02d" $P $STAGES)
+    sid=$(printf "%02d%02d%02d" $s $HL_RANDOM_DROPOUT $HL_BEAM_SIZE)
+    echo Producing featurization for ID: $pid $sid | tee -a $LOGFILE
+    $ADAMS2019_DIR/featurization_to_sample $PIPELINE_DIR/random_pipeline.featurization $t $pid $sid $PIPELINE_DIR/random_pipeline.sample
```

Comment:
This is the change to the call to featurization_to_sample that expects pipeline_id and sample_id, so I split them.

The behavior with the "onnx" string in the pipeline_id parameter did not seem to fail, but in trying to figure out how things work, it was easier for me to trace if I called it as in the source.